tests/cases/fourslash/codeFixClassImplementInterfaceMultipleMembersAndPunctuation.ts

```diff
@@ -27,13 +27,13 @@ class C1 implements I1 {
     x: number;
     y: number;
     z: number;
-    f() {
+    f(): number {
         throw new Error("Method not implemented.");
     }
-    g() {
+    g(): any {
         throw new Error("Method not implemented.");
     }
-    h() {
+    h(): any {

```

Comment:
This isn't a desirable side effect of the fix. We should still be having these unannotated (by moving the logic to `undefined`-out the return type of the method node to the calling code in the refactor)

Reply:
Resurrected `SuppressAnyReturnType` and made it used only at the toplevel.